web/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@thorvg/lottie-player",
+  "version": "1.0.0-beta.0",
+  "description": "A web lottie player using ThorVG as a renderer",
+  "main": "./dist/lottie-player.cjs.js",
+  "module": "./dist/lottie-player.esm.js",
+  "exports": {
+    ".": {
+      "import": "./dist/lottie-player.cjs.js",
+      "require": "./dist/lottie-player.esm.js"
+    }
+  },
```

```suggestion
  "exports": {
    ".": "./dist/lottie-player.js"
    "./*": "./dist/*.js"
  },
```

this change solves the TS error I am getting when I want to import `import { LottiePlayer } from '@thorvg/lottie-player';`

```
Could not find a declaration file for module '@thorvg/lottie-Lot player'. ".../node_modules/.ppm/@thorvg+lottie-player@1.0.0-beta.0/node_modules/@thorvg/lottie-player/dist/lottie-player cjs.js' implicitly has an 'any' type.
There are types at '...../app/node_modules/@thorvg/lottie-player/dist/lottie-player.d.ts', but this result could not be resolved when respecting package.json "exports". The 'Athorvg/lottie-player' library may need to update its package. json or typings. ts (7016)
```

---

`import { LottiePlayer } from '@thorvg/lottie-player';` is not proper import for this library, please use `import '@thorvg/lottie-player';`

---

could you please share the reason why? it is already possible, you are exporting the LottiePlayer class. Please check this Codepen example I created: https://codepen.io/aidosmf/pen/PoLYzVX Some devs 100% would like to utilize your library this way

The public methods are not working as well, ideally, it's better to update data using available methods instead of accessing the public variables